receiver/kymastatsreceiver/README.md

```diff
@@ -18,7 +18,7 @@ The following settings are required:
 
 - `auth_type` (default = `serviceAccount`): Specifies the authentication method for accessing the Kubernetes API server.
    Options include `none` (no authentication), `serviceAccount` (uses the default service account token assigned to the Pod), or `kubeConfig` (uses credentials from `~/.kube/config`).
-- `modules`: A list of API group-version-resources of Kyma modules. For each group-version-resource, the status metrics are generated for the first resource instance found.
+- `resources`: A list of API group-version-resources of Kyma resources. For each group-version-resource.
```

Comment:
the 2nd sentence seems to be missing a bit

Reply:
previously we were fetching status for only first resource. Now the behaviour has changed and we fetch for all the resources. So I removed the sentence

Reply:
Thank you! But what does the lonely “For each group-version-resource.” mean?
Does it now belong to the previous sentence? Do you want to say
“A list of API group-version-resources of Kyma resources for each group-version-resource.”
But then, what does that mean? Is there a list of g-v-resources for Kyma resources for each g-v-resource?